### DISCUSSION TITLE
Website Favicon Preview Unavailable on Mobile

### DISCUSSION THREAD

**Discussion by ghost** [colinwilson/lotusdocs#47] [Feature Requests]
### Past Issues Searched

- [X] I have searched open and closed issues to make sure that the bug has not yet been reported

### Issue is a Bug Report

- [X] This is a bug report and not a feature request, nor asking for support

### Describe the bug

When sharing a website link, the favicon image doesn't appear.

---

Steps to reproduce:

Type in the below message in WhatsApp (or any internet-based chat application) to see the website's favicon:
https://stackoverflow.com

Typing in the below message doesn't show the favicon:
https://lotusdocs.dev

---

Steps to solve:

Having the below files in `static/` is likely to make the preview available:

```
logo.svg                    # could be 512x512 or better
favicon.png                 # 32x32
favicon.ico                 # 32x32
apple-touch-icon.png        # 180x180
icon512.png                 # 512x512
```

### Expected behavior

For the favicon to be displayed when sending a message containing the link to the website.

**Comment by colinwilson** [comment]
This is related to [open graph](https://www.opengraph.xyz/). It's not yet coded but will be very soon. 👍
